core/cloudflow-spark-testkit/src/main/scala/cloudflow/spark/testkit/TestSparkStreamletContext.scala

```diff
@@ -44,7 +45,7 @@ private[testkit] class TestSparkStreamletContext(override val streamletRef: Stri
                                                  override val config: Config = ConfigFactory.empty)
     extends SparkStreamletContext(StreamletDefinition("appId", "appVersion", streamletRef, "streamletClass", List(), List(), config),
                                   session) {
-
+  val ProcessingTimeInterval = 1.second
```

Comment:
This is hardcoded that guesses that the data will be processed in 1 sec? I suspect that this is a bit fragile.

Reply:
No. This is the `Trigger` interval, not total processing time. I have observed that fixing the micro-batch period provides improved reliability in observing data being processed. Some tests had like a 40 seconds 'wait time' that can now be reduced to 2 seconds. (given that the fixed 1-second 'micro-batch' ensures that the data would have been processed already)

Reply:
I agree with @stavros, we should make it possible to change this.

Reply:
@RayRoestenburg I was typing the above msg before your comment. Does that answer your question as well?

Reply:
Thanks, yes.

Reply:
@skonto do you agree that this can stay as-is?